skyportal/tests/frontend/test_scanning_page.py

```diff
@@ -110,14 +110,6 @@ def test_candidate_unsaved_only_filtering(
         assert status == 200
         assert data["data"]["id"] == f"{candidate_id}_{i}"
 
-    status, data = api(
```

Comment:
Why delete this?

Reply:
this is totally unrelated, just cleaning up some unused extraneous code here

Reply:
probably should've just pushed this straight up to master